pkg/network/admitter/passt.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 Red Hat, Inc.
+ *
+ */
+
+package admitter
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
+func validatePasstBinding(
```

Comment:
Since passt iface API is deprecated in favor of Passt network binding plugin, I think it will be nice if this validation function would reflect it, and maybe recommend using the plugin.
Changing it on a follow up PR may be a better fit.

Reply:
This is part of the passt core binding check, not about the binding plugin which is not specific to passt or any other binding. When we remove passt core binding, we will remove these as well.